cheroot/test/test_core.py

```diff
@@ -273,6 +273,10 @@ def test_content_length_required(test_client):
             b'GET / HTTPS/1.1',  # invalid proto
             HTTP_BAD_REQUEST, b'Malformed Request-Line: bad protocol',
         ),
+        (
+            b'GET / HTTP/1',  # invalid version
```

Comment:
Ideas for extra PRs:
- `b'GET / HTTP/x'`
- `b'GET / HTTP/'`
- `b'GET / HTTP/\\0'`
- `b'GET / HTTP/☃'`